src/components/Chat/Chat.tsx

```diff
@@ -250,13 +274,45 @@ const ChatWithLLM: React.FC = () => {
                 className="h-full w-full m-x-auto color-gray-500 "
               />
             ) : (
-              <Button
-                className="bg-slate-700 w-[70px] border-none h-full hover:bg-slate-900 cursor-pointer text-center pt-0 pb-0 pr-2 pl-2"
-                onClick={handleSubmitNewMessage}
-                placeholder=""
-              >
-                Ask
-              </Button>
+              <>
+              <button  aria-expanded="false" aria-haspopup="menu"
+                className={
+                  `align-middle select-none font-sans font-bold transition-all 
+                  text-xs py-3 px-6 rounded-tl rounded-bl text-white shadow-md shadow-gray-900/10 
+                  hover:shadow-lg hover:shadow-gray-900/20 focus:opacity-[0.85] focus:shadow-none active:opacity-[0.85] 
+                  active:shadow-none bg-slate-700 border-none h-full hover:bg-slate-900 cursor-pointer text-center 
+                  pt-0 pb-0 pr-2 pl-2`} 
+                type="button"
+                onClick={handleSubmitNewMessage}>
+                  {askText}
+                </button>
+              <Menu placement="top-end">
+              <MenuHandler>
+              <button  aria-expanded="false" aria-haspopup="menu"
+                className={
+                  `align-middle select-none font-sans font-bold uppercase transition-all 
+                  text-xs py-3 px-6 rounded-tr rounded-br text-white shadow-md shadow-gray-900/10 
+                  hover:shadow-lg hover:shadow-gray-900/20 focus:opacity-[0.85] focus:shadow-none active:opacity-[0.85] 
+                  active:shadow-none bg-slate-700 border-none h-full hover:bg-slate-900 cursor-pointer text-center 
+                  pt-0 pb-0 pr-2 pl-2`} 
+                type="button">
+                  <div className="mb-1">⌄</div>
+              </button>
+              </MenuHandler>
+              <MenuList placeholder="" className="bg-slate-600">
+                {ASK_OPTIONS.map((option, index) => {
+                  return (
+                    <MenuItem key={index} placeholder="" 
+                      className="bg-slate-600 border-none h-full w-full hover:bg-slate-700 cursor-pointer text-white text-left" 
+                      onClick={() => setAskText(option)}>
+                        {option}
+                    </MenuItem>
```

Comment:
Now we have notes mode:
1. Ask specific note
2. Template prompts for those notes
3. Soon to directly connect with current file interfaced.

https://github.com/reorproject/reor/assets/58682208/1f6f2cc1-c853-4de6-a3f6-9fe7b8d5f6ae

Next up is allowing users to customize their own prompts

Reply:
nice ill add in the logic to filter for single file